### ISSUE TITLE
Not working in landscape on Galaxy S6

### ISSUE THREAD

**Issue by alexandre-g** [siebeprojects/samples-keyboardheight#1]
Only tested this phone, but `onKeyboardHeightChanged` isn't called at all when in landscape.

**Comment by FabianTerhorst** [comment]
You just used the unmodified sample project?

**Comment by alexandre-g** [comment]
Yep. The text view went full screen and maybe that's why. After I've added some flag to the EditText to prevent full screen in landscape mode, it was working from what I recall.

**Comment by ghost** [comment]
@alexandre-g What's your code?

**Comment by alexandre-g** [comment]
@blackberry2016 Don't have that code in project anymore, and can't remember, but I imagine it was this flag `android:imeOptions="flagNoExtractUi"`
